Dockerfile

```diff
@@ -19,32 +19,19 @@ ARG BEATS_VERSION=${BEATS_VERSION:-6.1.2}
 ARG GO_PLATFORM=${GO_PLATFORM:-linux-amd64}
 ARG AWSBEATS_VERSION=${AWSBEATS_VERSION:-1-snapshot}
 ARG BEAT_NAME=${BEAT_NAME:-filebeat}
+ARG GO_VERSION=${GO_VERSION:-1.10.2}
+ARG BEAT_GITHUB_REPO
+ARG BEAT_GO_PKG
+
 RUN curl --verbose --fail https://raw.githubusercontent.com/golang/dep/master/install.sh -o install.sh && sh install.sh && rm install.sh
-RUN go get github.com/elastic/beats || true
 RUN /go/bin/dep ensure
+RUN ls vendor/github.com
+RUN rm -rf vendor/github.com/elastic
```

Comment:
`dep ensure` introduced the elastic/beats pkg that was used only when building awsbeats, but beats.
I intentionally remove it, so that both awsbeats and beats are built using the same elastic/beats pkg installed into `$GOPATH/src/github.com/elastic/beats`, not the vendor dir of the project.